modules/@apostrophecms/widget-type/ui/apos/composables/AposWidget.js

```diff
@@ -80,8 +80,15 @@ export async function _renderContent(props) {
     }
     return {};
   } catch (e) {
+    const errData = {
+      widgetTitle: widget.title,
+      widgetType: props.type,
+      documentId: props.docId,
+      errors: e.body.data?.errors || []
```

Comment:
Sorry I missed it the first time. I don't think we can be absolutely sure that `e.body` will be an object. I'll do `e.body?.data?.errors` instead